pcdsdevices/slits.py

```diff
@@ -114,23 +114,19 @@ class Slits(Device, MvInterface):
     make a rough back of the hand calculation without being over aggressive
     about changing slit widths during alignment
     """
-    xcenter = C(SlitPositioner, '', slit_type="XCENTER")
-    xwidth = C(SlitPositioner, '', slit_type="XWIDTH")
-    ycenter = C(SlitPositioner, '', slit_type="YCENTER")
-    ywidth = C(SlitPositioner, '', slit_type="YWIDTH")
-    blocked = C(EpicsSignalRO, ":BLOCKED")
-    open_cmd = C(EpicsSignal, ":OPEN")
-    close_cmd = C(EpicsSignal, ":CLOSE")
-    block_cmd = C(EpicsSignal, ":BLOCK")
+    xcenter = C(SlitPositioner, '', slit_type="XCENTER", kind='hinted')
+    xwidth = C(SlitPositioner, '', slit_type="XWIDTH", kind='config')
+    ycenter = C(SlitPositioner, '', slit_type="YCENTER", kind='hinted')
+    ywidth = C(SlitPositioner, '', slit_type="YWIDTH", kind='config')
```

Comment:
I feel like we want the widths to be `normal`, because you want to know if they have changed in the data